src/entries/popup/pages/home/TabHeader.tsx

```diff
@@ -86,7 +86,7 @@ export function TabHeader({
         display="flex"
         justifyContent="space-between"
         paddingHorizontal="20px"
-        style={{ maxHeight: 11, textTransform: 'capitalize' }}
+        // style={{ maxHeight: 11, textTransform: 'capitalize' }}
```

Comment:
👀 was this maxHeight breaking UI when the testnet bar is showing? Would prefer to keep it if we can.

Reply:
rolling back

Reply:
oh ok rolling back